src/test/java/net/snowflake/client/jdbc/PreparedStatement1LatestIT.java

```diff
@@ -239,6 +240,35 @@ public void testSetObjectMethodWithVarbinaryColumn() throws SQLException {
     }
   }
 
+  @Test
+  public void testSetObjectMethodWithBigIntegerColumn() throws SQLException {
+    try (Connection connection = init()) {
+      connection.createStatement().execute("create or replace table test_bigint(id NUMBER)");
+
+      try (PreparedStatement prepStatement =
```

Comment:
Do you want to catch an exception and gracefully raise it as an error here?

Reply:
@sfc-gh-hchaturvedi fixed, thanks!